### PR TITLE
Small Apache 2.2 fixes

### PR DESCRIPTION
**Pull request by bmw**
Fixes:
* Trying to modify the wrong config file in CentOS6.
* Not adding `NameVirtualHost` causing `_default_` and `*` conflicts.

### CODE REVIEW

File: letsencrypt-apache/letsencrypt_apache/configurator.py [hunk 1]
```diff
@@ -874,9 +874,18 @@ def _add_name_vhost_if_necessary(self, vhost):
...
         # Remember 1.1.1.1:* == 1.1.1.1 -> hence any()
         for addr in vhost.addrs:
+
+            # In Apache 2.2, when a NameVirtualHost directive is not
+            # set, "*" and "_default_" will conflict when sharing a port
+            addrs = [addr]
+            if addr.get_addr() == "*":
+                addrs.append(obj.Addr(("_default_", addr.get_port(),)))
+            elif addr.get_addr() == "_default_":
+                addrs.append(obj.Addr(("*", addr.get_port(),)))
```
Comment (pde) [inline letsencrypt-apache/letsencrypt_apache/configurator.py]: This is shorter, not sure if it's better:
```
if addr.get_addr() in ("*", "_default_"):
    addrs = [obj.Addr(a, addr.get_port()) for a in ("*", "_default_")]
```

**Comment by pde** [comment]
LGTM

File: letsencrypt-apache/letsencrypt_apache/configurator.py (hunk 1 above)
Comment (bmw) [inline letsencrypt-apache/letsencrypt_apache/configurator.py]: I think it's better. Thanks for the suggestion. I made the change.

**Comment by pde** [comment]
Also LGTM

**Pull request merged**